### ISSUE TITLE
Simultaneous acquisition from multiple cameras does not work

### ISSUE THREAD

**Issue by dmgav** [areaDetector/ADVimba#8]
Is it possible to run acquisition from multiple cameras using one server with multiple IOCs installed?

We are setting up multiple Maco 319C cameras for monitoring equipment. The cameras are connected to the local network. All IOCs (ADVimba) are installed on a single server, which is also connected to the network. The acquisition works well with each individual camera (Acquire is set to 1 only on one camera IOC) reaching the rate over 35 frames per second. Acquisition stops on all cameras if Acquire is set to 1 for the second camera (two or more cameras are activated simultaneously). Multiple cameras can run simultaneously if acquisition period is large (0.5 or 1 second), but acquisition fails quickly if the rate increased.

**Comment by MarkRivers** [comment]
Have you changed the system network buffering settings as described here:
https://areadetector.github.io/areaDetector/ADGenICam/ADGenICam.html#linux-usb-and-gige-system-settings

You should be able to run multiple cameras close to the limit of the total network bandwidth.

We run 3 IOCs on a Windows machine, with each camera using a dedicated NIC on the PC.  The cameras can all run at their full rate.

**Comment by MarkRivers** [comment]
@dmgav were you able to resolve this issue?

**Comment by bsobhani** [comment]
I had a similar issue where acquiring from two cameras on the same computer resulted in most frames being dropped. At the advice of Vimba tech support I lowered the cameras' StreamBytesPerSecond settings from 115000000 to 55000000 and then the cameras stopped dropping frames. I think it may be because two cameras at 115MBytes each may exceed my switch's 1GBit limit.

**Comment by MarkRivers** [comment]
Yes, if you run 2 cameras on the same NIC you need to either manually configure the frame rates so you don't exceed the 1 Gbit combined bandwidth limit, or use StreamBytesPerSecond to prevent accidentally requesting a higher rate.

**Comment by bsobhani** [comment]
When StreamBytesPerSecond was 55000000, the ArrayRate_RBV PV showed it was acquiring at 12Hz for both cameras, and no frames were being dropped. If I ensure the exposure time and acquire period in the IOC are 12HZ for both cameras, but then change the StreamBytesPerSecond back to 115000000, then frames start getting dropped again.

**Comment by MarkRivers** [comment]
OK, that is interesting and not clear why that happens since the network bandwidth should be the same in both cases.

**Comment by MarkRivers** [comment]
Since @dmgav has not responded to my question for more than 8 months I am closing this issue.

**Comment by dmgav** [comment]
I apologize, I was able to resolve the issue following the instructions. Thank you.
